r-build.md

```diff
@@ -0,0 +1,177 @@
+started, the job has a time limit of four hours.
+
+## 3. Compile Base R From Source Code
+
+Once the interactive job starts, R can be compiled. The general workflow will be 
+to first run the R configuration script followed by `make` and `make install`. 
+
+To install R, first create a directory that will hold your R installation (e.g. `R/`) 
+, then extract all of the source code files from R-#.#.#.tar.gz and move into the source code directory:
+
+``` 
+[alice@build]$ mkdir R/
+[alice@build]$ tar xzf R-#.#.#.tar.gz
+[alice@build]$ cd R-#.#.#/
+```
+{:.term}
+
+From the source code directory, run the following commands. **The `make` step may take a while to 
+complete!**: 
+
+```
+[alice@build]$ ./configure --prefix=$_CONDOR_SCRATCH_DIR/R
+[alice@build]$ make
+[alice@build]$ make install
+```
+{:.term}
+
+Then return the the top level directory:
+
+```
+[alice@build]$ cd ../
+```
+{.term}
+
+If R has been installed successfully, you should be able to run the following command as a test:
+
+```
+[alice@build]$ R/bin/R --version
+```
+{.term}
+
+The output of the above command should match the version number that you just installed!
+
+## 4. Install Packages 
+
+After your R compilation has completed, if any additional R packages are required for your work, 
+please follow the steps in our [R Jobs](http://chtc.cs.wisc.edu/r-jobs.shtml) guide 
+starting with the `export` commands shown in step 
+[1.B.1](http://chtc.cs.wisc.edu/r-jobs.shtml#b.-install-the-packages). After creating a compressed tar 
+archive of your `packages` directory return the the next steps in this guide.
+
+## 5. Create A Portable Copy of Your R Installation
+
+Once R has been successfully compiled (and after any packages have been installed), the final step 
+is to create a compressed tar archive of your R installation. From the top level directory:
+
+```
+[alice@build]$ tar czf my_R#.#.#.tar.gz R/
+```
+{.term}
+
+**Be sure to check the size of your R tar archive:**
+```
+[alice@build]$ ls -lh my_R#.#.#.tar.gz
+```
+{.term}
+
+**If the size of `my_R#.#.#.tar.gz` is larger than ~100MB then your will need to use 
+the CHTC Squid Wed Proxy to host this file.** See our 
+[Squid Web Proxy](http://chtc.cs.wisc.edu/file-avail-squid.shtml) guide for more details.
+
+Now you have a portable copy of your R installation that can be brought along with your jobs. 
+The final step is to terminate your interactive job, and HTCondor will tranfer your R tar archive 
+back to your `/home` directory:
+
+```
+[alice@submit]$ exit
+```
+{.term}
+
+Once your interactive job terminates, you will be back in your `/home` directory on the submit 
+server and should see a copy of your R tar archive:
+
+```
+[alice@submit]$ ls
+```
+{.term}
+
+## 6. Use Your Custom R Installation In Your Jobs
+
+To use the R installation that was built using the steps in this guide, please follow 
+the steps and examples described on our [R Jobs](http://chtc.cs.wisc.edu/r-jobs.shtml) guide.
```

Comment:
Add something like "but replace `transfer_input_files = http://proxy.chtc.wisc.edu/SQUID/chtc/R###.tar.gz` with `transfer_input_files = ...`" to highlight where they do have to change something.